### DISCUSSION TITLE
Graylog-Ansible 4.2

### DISCUSSION THREAD

**Discussion by moneyRunner23** [Graylog2/graylog-ansible-role#196] [Q&A] [answered]
Does anybody know where the corresponding releases for Graylog 4 went? They're referenced in our repo from a few years back, but do not seem to exist in this location anymore.

**Comment by bernd** [comment, marked as answer]
@moneyRunner23 Can you please elaborate? What exactly is missing?

The role versions don't relate to Graylog versions if that's what you mean.

**Reply by moneyRunner23** [reply]
Thanks, bernd. I got back to to work on my issue and got past the hurdle.
